ironfish-rust/src/transaction/tests.rs

```diff
@@ -837,3 +839,31 @@ fn test_aggregate_signature_shares() {
     // verify transaction
     verify_transaction(&signed_transaction).expect("should be able to verify transaction");
 }
+
+#[test]
+fn test_add_signature() {
```

Comment:
Why are you using hex repr here? What you are really trying to test here is adding signature to a unsigned transaction, irrespective of ledger. You should be constructing the transaction in code in the test.

Reply:
aggregate_signature_shares tests should cover this now